### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by mwismann**
# Vet clinic database: query and update animals table

**The following changes were implemented:**

### On `data.sql`: 

- Inserted new pieces of data ✅ 

### On `queries.sql`:

- Updated the `animals` table by setting the `species` column to `digimon` for all animals that have a name ending in `mon` ✅
check_1

- Updated the `animals` table by setting the `species` column to `pokemon` for all animals that don't have species already set ✅ 
check_2

- Verified that changes persisted after commit ✅ 
check_3

- Deleted all records from animals inside a transaction, then rolled back and checked the changes were unmade ✅ 
check_4

- Inside another transaction: deleted all animals born after Jan 1st, 2022, created a savepoint for the transaction, updated all animals' weight to be their weight multiplied by -1, rolled back to the savepoint, updated all animals' weights that are negative to be their weight multiplied by -1, and committed the transaction ✅ 
check_5

### On `queries.sql`:

- Wrote queries for the following quests:

1. How many animals are there?
query_1

2. How many animals have never tried to escape?
query_2

3. What is the average weight of animals?
query_3

4. Who escapes the most, neutered or not neutered animals?
query_4

5. What is the minimum and maximum weight of each type of animal?
query_5

6. What is the average number of escape attempts per animal type of those born between 1990 and 2000?
query_6

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -4,3 +4,10 @@ INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_k
...
 INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (3, 'Pikachu', '2021-01-07', 1, false, 15.04);
 INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (4, 'Devimon', '2017-05-12', 5, true, 11);
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (5, 'Charmander', '2020-02-08', 0, false, -11);
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (6, 'Plantmon', '2021-11-15', 2, true, -5.7);
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (7, 'Squirtle', '1993-04-02', 3, false, -12.13);
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (8, 'Angemon', '2005-06-12', 1, true, -45);
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (9, 'Boarmon', '2005-06-07', 7, true, 20.4);
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (10, 'Blossom', '1998-11-13', 3, true, 17);
```
Comment (zieeco) [inline data.sql]: - Hey, let's correct the date to ` '1998/10/13'`

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @mwismann,




Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice updating the table. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions
_Check the comments under the review._

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (mwismann) [reply data.sql]: Hey @zieeco, thank you for noticing the error.

**Review by fobadara — APPROVED** [review]
Hi @mwismann, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

APPRVED
## Highlights
✅ Descriptive README
✅ Descriptive PR summary
✅ Clean code



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**